apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -104,6 +105,52 @@ describe('Delegation Controller', () => {
       });
   });
 
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid accountId', async () => {
+    const providerId = provider.msaId?.toString();
+    const { keypair } = users[1];
+    const invalidAccountId = `${keypair.address.slice(0, -1)}5`;
+    const getPath: string = `/v1/delegation/revokeDelegation/${invalidAccountId}/${providerId}`;
+    await request(httpServer).get(getPath).expect(400).expect({
+      statusCode: 400,
+      message: 'Invalid accountId',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with valid accountId: no msa', async () => {
+    const providerId = provider.msaId?.toString();
+    const keyring = new Keyring({ type: 'sr25519' });
+    const keypair = keyring.addFromUri('//Bob');
+
+    const getPath: string = `/v1/delegation/revokeDelegation/${keypair.address}/${providerId}`;
+    await request(httpServer).get(getPath).expect(404).expect({
+      statusCode: 404,
+      message: 'MSA ID for account not found',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid providerId', async () => {
+    const { keypair } = users[1];
+    const accountId = keypair.address;
+    const invalidProviderId = '9999';
```

Comment:
Maybe use an MSA from the provisioned `users[]` array here; they're guaranteed not to be registered Providers. And if you're looking for a value that's guaranteed not a registered MSA, you can query `api.query.msa.currentMsaIdentifierMaximum()` and add > 1 to it.